src/components/employee-beta/forms/EmployeeForm.vue

```diff
@@ -104,7 +104,7 @@
                 @click="cardName = 'Customer Orgs'"
                 link
                 title="Customer Orgs"
-                :class="{ invalid: !validTabs.customerOrgs }"
+                :class="{ invalid: !validTabs.customerOrg === false }"
```

Comment:
```suggestion
                :class="{ invalid: validTabs.customerOrg === false }"
```
I feel like invalid should be when validTabs for customerOrg is false (matches what the other ones are)